examples/ng_6lo_udp/Makefile

```diff
@@ -0,0 +1,29 @@
+# name of your application
+APPLICATION = ng_6lo_udp
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Specify the modules that are used
+USEMODULE += ng_netif
+USEMODULE += ng_sixlowpan
```

Comment:
no, why should it be dependent on the device? What if I want to use the radio without 6LoWPAN (which is a valid use-case...)?